src/lib/core/datetime.h

```diff
@@ -146,6 +146,102 @@ datetime_parse_full(struct datetime *date, const char *str, size_t len,
 size_t
 datetime_strptime(struct datetime *date, const char *buf, const char *fmt);
 
+/** Return the year from a date. */
+int64_t
+datetime_year(const struct datetime *date);
+
+/** Return the month of year from a date. */
+int64_t
+datetime_month(const struct datetime *date);
+
+/** Return the quarter of year from a date. */
+int64_t
+datetime_quarter(const struct datetime *date);
+
+/** Return the week of year from a date. */
+int64_t
+datetime_week(const struct datetime *date);
+
+/** Return the day of month from a date. */
+int64_t
+datetime_day(const struct datetime *date);
+
+/** Return the day of week from a date. */
+int64_t
+datetime_dow(const struct datetime *date);
+
+/** Return the day of year from a date. */
+int64_t
+datetime_doy(const struct datetime *date);
+
+/** Return the hour from a date. */
+int64_t
+datetime_hour(const struct datetime *date);
+
+/** Return the hour of day from a date. */
+int64_t
+datetime_min(const struct datetime *date);
```

Comment:
The comment seems to be out of date.

Reply:
Thanks, fixed this comment and a few others below.